src/main.js

```diff
@@ -68,34 +77,42 @@ function clearInputFields() {
   buttonValidity();
 }
 
-ideaCard.addEventListener('click', function(event) {
-  favoriteIdeaCard(event);
-  deleteIdeaCard(event);
-});
 
 function deleteIdeaCard(event) {
   var closestIdea = event.target.closest('article');
   if (event.target.id === 'deleteIcon') {
-    for (var i = 0; i < ideaCardList.length; i++) {
-      if (ideaCardList[i].id === Number(closestIdea.id)) {
-        ideaCardList.splice(i, 1);
+    for (var i = 0; i < ideaCardListMain.length; i++) {
+      if (ideaCardListMain[i].id === Number(closestIdea.id)) {
+        ideaCardListMain.splice(i, 1);
       }
     }
   }
-  renderIdeaCard();
+  renderIdeaCard(ideaCardListMain);
 }
 
 function favoriteIdeaCard(event) {
   var closestIdea = event.target.closest('article');
-  if (event.target.id === 'favoriteIcon');
-    for (var i = 0; i < ideaCardList.length; i++) {
-      if (ideaCardList[i].id === Number(closestIdea.id)) {
-        console.log('where you', ideaCardList[i]);
-        ideaCardList[i].updateIdea();
+  if (event.target.id === 'favoriteIcon') {
+    for (var i = 0; i < ideaCardListMain.length; i++) {
+      if (ideaCardListMain[i].id === Number(closestIdea.id)) {
+        ideaCardListMain[i].updateIdea();
       }
     }
+  }
 }
 
+function searchIdea() {
```

Comment:
we should remove the console.log from this function 
otherwise looks sweet!! gold stars